modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting RolloutManager custom resource
+
+Uninstallation of {gitops-title} Operator does not remove any resources it created. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the RolloutManager custom resource (CR) and then uninstall the {gitops-title} Operator.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0, or a later version, is installed in your cluster.
+* RolloutManager custom resources are available in the namespace of your choice.
+
+.Procedure 
+
+. Log in to the {product-title} web console. 
+
+. Click *Operators* -> *Installed Operators*.
+
+. Create or select the project where RolloutManager CR for the progressive delivery is setup from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the Tabs section, click *RolloutManager* to find the list of RolloutManager instances under the *RolloutManagers* section.
+
+. Select the instance by clicking on its name. 
+
+. At the top right corner of the pane, click *Actions* -> *Delete RolloutManager* from the drop-down menu, and click *Delete* to confirm in the dialog box.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the RolloutManager instance is not available anymore.
```

Comment:
object formatting

Reply:
@abrennan89 Here it is not the API object but an instance of RollloutManager in the UI. Similar to an ArgoCD instance in the UI. The name of this instance is customizable. So we have to treat it the same way as we write for an ArgoCD instance. So I am leaving it as is.